docs/reference/metadata.md

```diff
@@ -194,13 +190,13 @@ Subfields:
 
 A `person_or_org` is described with the following subfields:
```

Comment:
I am not sure why GitHub's syntax highlighting is broken here. Everything renders nicely locally and on the web clearly...